GAMEFLOW.md

```diff
@@ -0,0 +1,1306 @@
+    </td>
+    <td>Integer</td>
+    <td>Exits to the specified cinematic level.</td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>exit_to_level</code>
+    </td>
+    <td>
+      <code>level_id</code>
+    </td>
+    <td>Integer</td>
+    <td>Exits to the specified level.</td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>exit_to_title</code>
+    </td>
+    <td colspan="2" align="center">N/A</td>
+    <td>Returns to the title level.</td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>flip_map</code>
+    </td>
+    <td colspan="2" align="center">N/A</td>
+    <td>Triggers the flip map.</td>
+  </tr>
+  <tr valign="top">
+    <td rowspan="2">
+      <code>give_item</code>
+    </td>
+    <td>
+      <code>object_id</code>
+    </td>
+    <td>Integer</td>
+    <td rowspan="2">
+      Adds the specified item and quantity to Lara's inventory.
+    </td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>quantity</code>
+    </td>
+    <td>Integer</td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>level_stats</code>
+    </td>
+    <td>
+      <code>level_id</code>
+    </td>
+    <td>Integer</td>
+    <td>Displays the end of level statistics for the given level number.</td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>loop_cine</code>
+    </td>
+    <td colspan="2" align="center">N/A</td>
+    <td>Plays the cinematic loop.</td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>loop_game</code>
+    </td>
+    <td colspan="2" align="center">N/A</td>
+    <td>Plays the main game loop.</td>
+  </tr>
+  <tr valign="top">
+    <td rowspan="3">
+      <code>mesh_swap</code>
+    </td>
+    <td>
+      <code>object1_id</code>
+    </td>
+    <td>Integer</td>
+    <td rowspan="3">Swaps the given mesh ID between the given objects.</td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>object2_id</code>
+    </td>
+    <td>Integer</td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>mesh_id</code>
+    </td>
+    <td>Integer</td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>play_fmv</code>
+    </td>
+    <td>
+      <code>fmv_path</code>
+    </td>
+    <td>String</td>
+    <td>Plays the specified FMV file.</td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>play_synced_audio</code>
+    </td>
+    <td>
+      <code>audio_id</code>
+    </td>
+    <td>Integer</td>
+    <td>Plays the given audio track.</td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>remove_ammo</code>
+    </td>
+    <td colspan="2" align="center">N/A</td>
+    <td>
+      Removes all ammo from Lara's inventory (guns and medipacks are retained).
+      If present, this must appear before the <code>start_game</code> sequence.
+    </td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>remove_guns</code>
+    </td>
+    <td colspan="2" align="center">N/A</td>
+    <td>
+      Removes all weapons from Lara's inventory. If present, this must appear
+      before the <code>start_game</code> sequence.
```

Comment:
Maybe mention Lara retains the remove gun's ammo?

Reply:
Thanks. I've reworked this bit of the table to avoid too much repetition and expanded on it further too.